addons/mail/static/src/new/composer/composer_view.js

```diff
@@ -0,0 +1,153 @@
+/** @odoo-module */
```

Comment:
i dont like renaming the component. it's so much simpler to have a composer component